### PR TITLE
Add REST API information.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Add REST API information. And fixed some minor CLI issues.

### CODE REVIEW

File: docs/Reference/Rest_API/Rest.md [hunk 1]
```diff
@@ -4,11 +4,44 @@ description: Teku RESTful APIa
...
+The Swagger UI endpoint can be accessed at `http:<interface>:<port>/swagger-ui` where:
+
+* `interface` is specified using [`--rest-api-interface`](../CLI/CLI-Syntax.md#rest-api-interface)
+* `port` is specified using [`--rest-api-port`](../CLI/CLI-Syntax.md#rest-api-port)
+
+The default location is `http://localhost:5051/swagger-ui`.
+
+## REST API endpoints
+
 The RESTful JSON HTTP API provides access to beacon chain, node, and network
```
Comment (macfarla) [inline docs/Reference/Rest_API/Rest.md:29]: feels like this sentence is repeating what's in the table below but missing some info. maybe we can remove or generalise this sentence?

File: docs/Reference/Rest_API/Rest.md [hunk 2]
```diff
@@ -4,11 +4,44 @@ description: Teku RESTful APIa
...
+to interact with the APIs using Swagger UI.
+
+The [Teku REST API documentation] is also available to view information about the REST APIs.
+
+## Enable the REST API
+
+Enable the REST API service by setting the [`--rest-api-enabled`](../CLI/CLI-Syntax.md#rest-api-enabled)
+command line option to `true`.
+
+You can enable the Swagger UI endpoint to visually interact with the APIs by setting the
```
Comment (macfarla) [inline docs/Reference/Rest_API/Rest.md:17]: visually doesn't seem like the right word. interact using OpenAPI framework? Or do we need to explain - normal interaction can be done via curl but we also have this other way of doing it via the web browser

File: docs/Reference/Rest_API/Rest.md (hunk 1 above)
Comment (bgravenorst) [reply docs/Reference/Rest_API/Rest.md:29]: Yes I agree. Best to remove this.

File: docs/Reference/Rest_API/Rest.md [hunk 3]
```diff
@@ -4,11 +4,47 @@ description: Teku RESTful APIa
...
+the API documentation endpoint to interact with APIs using Swagger UI.
+
+[Teku REST API documentation] is also available.
+
+## Enable the REST API service
+
+Enable the REST API service by setting the [`--rest-api-enabled`](../CLI/CLI-Syntax.md#rest-api-enabled)
+command line option to `true`.
+
+You can interact with Teku APIs using the web browser by setting the
```
Comment (MadelineMurray) [inline docs/Reference/Rest_API/Rest.md:17]: I think we could drop _You can_ here to make this more active. ie, 
Interact with Teku APIs ....

File: docs/Reference/Rest_API/Rest.md (hunk 1 above)
Comment (bgravenorst) [reply docs/Reference/Rest_API/Rest.md]: Done

File: docs/Reference/Rest_API/Rest.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/Rest_API/Rest.md]: Done

File: docs/Reference/Rest_API/Rest.md (hunk 3 above)
Comment (bgravenorst) [reply docs/Reference/Rest_API/Rest.md:17]: Done

**Pull request merged**